example/rely/TestFramework.re

```diff
@@ -0,0 +1,19 @@
+let projectDir =
```

Comment:
would it be simpler to run tests with an environment variable pointing to the project root?